src/App.sass

```diff
@@ -1,8 +1,7 @@
 @charset "utf-8";
 
 // Import a Google Font
-@import url('https://fonts.googleapis.com/css2?family=Poppins:400,700');
-
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&family=Roboto+Mono:wght@400;700&display=swap');
```

Comment:
Last thing: if you want to use Roboto as a font on your site, you need to import it *in addition* to Roboto Mono. The two are distinct font families. However, if you use Roboto, then Poppins might be redundant since they are both sans serif fonts. I'll let you make the decision on this, but just make sure you use the minimum number of fonts.